.circleci/config.yml

```diff
@@ -818,6 +824,31 @@ jobs:
           path: builds-test-mmi-playwright
           destination: builds-test-mmi-playwright
 
+  prep-build-test-swap-playwright:
```

Comment:
Could you elaborate on why we need a new build here? We have been spending way too much CircleCI credits recently and have been trying to cut costs in any way we can, such as by removing builds and workflows. But we are adding one here.

Reply:
For example, could this new test use the pre-existing production or test build? I would expect playwright tests to generally use the test build, though this one seems to use one equivalent to the production build.

Reply:
@Gudahtt @dan437 I have found a pre-existing production build that works so we don't have to generate a new one. Please review again